app/tests/unit/components/ProjectRevision/routinelyUpdatedFieldFactory.test.tsx

```diff
@@ -219,3 +219,30 @@ describe("The routinely updated field factory", () => {
     expect(logSpy).toHaveBeenCalledWith("update mutation called");
   });
 });
+
+it("should return 'no options' when no schema.anyOf does not exist or is empty", async () => {
```

Comment:
We're not doing routinely updated fields anymore, so I believe we'll be removing this file at some point as tech debt (@Sepehr-Sobhani, can you confirm?). I'd suggest moving this test into `SearchDropdown.test.tsx` (and changing the filename to `SearchDropdownWidget.test.tsx` if you're already in there)

Reply:
Yes, we no longer need `routinelyUpdatedFieldFactory` and its related test.

Reply:
Good catch! Thanks for clarifying :)